fhir-bulkimportexport-webapp/src/main/java/com/ibm/fhir/bulkimport/ImportJobListener.java

```diff
@@ -60,6 +77,30 @@ public void afterJob() {
             return;
         }
 
+        // Generate import summary and pass it into ExitStatus of the job execution.
+        // e.g, [3:0, 4:1] means 3 resources imported and 0 failures for the 1st file, and 4 imported and 1 failure for the 2nd file.
+        JsonReader reader = Json.createReader(new StringReader(
```

Comment:
82-85 is repeated in many locations.  It seems that it is ideal for a small utility method shared across Jobs. and could use try-catch-close

Reply:
yes,it's used by both listener and mapper. let me create a util function for it.